src/history.js

```diff
@@ -297,6 +304,14 @@ class History extends Emitter {
 
     this.size = size
   }
+
+  /**
+   * Determine if provided action is within active history branch
+   * @param {Action} action
+   */
+  isActive(action) {
+    return this.toArray().indexOf(action) >= 0
```

Comment:
Just a stray, mostly inactionable thought, but I wonder if we could design action ids to be able to tell if one action proceeds another in the same branch of history. 

Stray thought. This is great, and very easy to understand.